org.eclipse.xtext.xtext.generator/src/org/eclipse/xtext/xtext/generator/ui/contentAssist/ContentAssistFragment2.xtend

```diff
@@ -159,11 +160,25 @@ class ContentAssistFragment2 extends AbstractInheritingFragment {
 		// take the non-abstract class signature for the src-gen class in case of !generateStub
 		//  as proposalProviders of sub languages refer to 'grammar.proposalProviderClass',
 		//  see 'getGenProposalProviderSuperClass(...)'
+		generateGenJavaProposalProvider(assignments, remainingRules)
```

Comment:
While we are at it: does it make sense to extract methods `getAssignments` and `getRemainingRules`?